plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -78,7 +88,12 @@ library PaymentProcessInFlightExit {
 
                 if (shouldWithdrawOutput(self, exit, withdrawal, token, i)) {
                     withdrawFromVault(self, withdrawal);
-                    withdrawal.exitTarget.transfer(withdrawal.piggybackBondSize);
+                    // solhint-disable-next-line avoid-call-value
+                    (bool success, ) = withdrawal.exitTarget.call.value(withdrawal.piggybackBondSize)("");
+                    // we do not want to block a queue if bond return is unsuccessful
+                    if (!success) {
+                        emit InFlightBondReturnFailed(withdrawal.exitTarget, withdrawal.piggybackBondSize);
```

Comment:
Does that mean that the funds get stuck in the contract forever?

Reply:
yes